# Clase 3

#### Ejercicio 3 (No-Tarea 2)

$$ \dfrac{(6\times10^3)(7\times10^{-10})}{(2\times10^2)^2}-3.21\times10^{-2} $$

#### Solución

Por la jerarquía de operaciones sabemos que lo primero que tenemos que hacer es resolver potencias. Para esto usamos la ley de exponentes: $(a\cdot b)^n=a^n\cdot b^n$

$$ \dfrac{(6\times10^3)(7\times10^{-10})}{2^2\times(10^2)^2}-3.21\times10^{-2} $$

Ahora usamos la ley de exponentes: $(a^m)^n=a^{mn}$

$$ \dfrac{(6\times10^3)(7\times10^{-10})}{4\times10^4}-3.21\times10^{-2} $$

Recuerden que si tenemos cosas multiplicándose y dividiéndose, podemos acomodar como queramos (de preferencia, como nos resulte conveniente).

$$ \dfrac{6\cdot7}{4}\times\dfrac{10^3\cdot10^{-10}}{10^4}-3.21\times10^{-2} $$

$$ \dfrac{42}{4}\times\dfrac{10^{-7}}{10^4}-3.21\times10^{-2} $$

$$ 10.5\times10^{-11}-3.21\times10^{-2} $$

$$ 1.05\times10^{-10}-3.21\times10^{-2} $$

En el penúltimo paso usamos la ley de exponentes: $\dfrac{a^m}{a^n}=a^{m-n}$

Finalmente cambiamos el más pequeño de los números (aquel que tenga el exponente más bajo) al orden de magnitud del otro. Como para pasar de $-10$ a $-2$ hay que sumarle $8$ al exponente, tendremos que recorrer el punto ocho veces hacia la izquierda, es decir, 
$$ 1.05\times10^{-10}=0.0000000105\times10^{-2}$$

Entonces, nos queda

$$0.0000000105\times10^{-2}-3.21\times10^{-2}$$

Para esto podemos usar la calculadora de la compu, que nos da

In [1]:
0.0000000105-3.21

-3.2099999895

Por lo tanto, el resultado del ejercicio es

$$ -3.2099999895\times10^{-2} $$

## Álgebra

### Productos notables

**Binomio al cuadrado**

$$ (x+y)^2=(x+y)(x+y)=x^2+xy+xy+y^2=x^2+2xy+y^2 $$

*"El resultado del binomio el cuadrado es el primer término al cuadrado más el doble del producto del primer término por el segundo término más el segundo término al cuadrado."*

* ***Ejemplo:***

$$ (3a^2y+5xy)^2 $$

$$ =(3a^2y)^2+2(3a^2y)(5xy)+(5xy)^2 $$

$$ =9a^4y^2+30a^2xy^2+25x^2y^2 $$

* ***Ejemplo:***

$$ (10ay^6-7x^3y^8)^2 $$

$$ =(10ay^6)^2+2(10ay^6)(-7x^3y^8)+(-7x^3y^8)^2 $$

$$ =(10)^2(a)^2(y^6)^2+2(10)(-7)ax^3y^6y^8+(-7)^2(x^3)^2(y^8)^2 $$

$$ =100a^2y^{12}-140ax^3y^{14}+49x^6y^{16} $$

**Diferencia de cuadrados**

$$ (x+y)(x-y) $$

$$ x^2-xy+xy-y^2 $$

$$ x^2-y^2 $$

*"Al multiplicar un binomio por su conjugado (o sea, el que tiene el signo contrario) lo que obtenemos es el primer término al cuadrado menos el segundo término al cuadrado."*

* ***Ejemplo:***

$$ (5ab^2+3a^2b^3)(5ab^2-3a^2b^3) $$

$$ (5ab^2)^2-(3a^2b^3)^2 $$

$$ 25a^2b^4-9a^4b^6 $$

## Despejes

*Despejar* significa dejar solo a algo. Las operaciones pasan como la operación inversa del otro lado de la ecuación.

$$ +\quad\leftrightarrow\quad- $$

$$ \cdot\quad\leftrightarrow\quad\div $$

$$ {}^2\quad\leftrightarrow\quad\sqrt[2]{} $$

$$ {}^3\quad\leftrightarrow\quad\sqrt[3]{} $$

¿Cómo **rayos** despejamos? Siguiendo la jerarquía al revés. Pasamos del otro lado dela ecuación las operaciones respecto al término que contenga a lo que queremos despejar.

* ***Ejemplo:***

Despejar $y$

$$ 4(3x-a+5y^2)-32=0 $$

$$ 4(3x-a+5y^2)=0+32 $$

$$ (3x-a+5y^2)=+\frac{32}{4} $$

$$ 3x-a+5y^2=+8 $$

$$-a+5y^2=8-3x $$

$$ 5y^2=8-3x+a $$

$$ y^2=\frac{8-3x+a}{5} $$

$$ y=\sqrt{\frac{8-3x+a}{5}} $$

* ***Ejemplo:***

Despejar $t$

$$ v=\frac{d}{t} $$

$$ vt=d $$

$$ t=\frac{d}{v} $$

* ***Ejemplo:***

Despejar $w$. (Cuando tenemos lo que queremos despejar en ambas partes de una fracción conviene pasar el denominador del otro lado.)

$$ y=\frac{wz+tx^2}{x+w+1} $$

$$ y=\frac{wz+tx^2}{x+w+1} $$

$$ y(x+w+1)=wz+tx^2 $$

$$ xy+wy+y=wz+tx^2 $$

$$ xy+wy+y-wz=tx^2 $$

$$ wy+y-wz=tx^2-xy $$

$$ wy-wz=tx^2-xy-y $$

$$ w(y-z)=tx^2-xy-y $$

$$ w=\frac{tx^2-xy-y}{y-z} $$

* ***Ejemplo:***

$$ 36xy-70x^2=9 $$

¿Qué pasa si pasó el 9 de la derecha dividiendo?

$$ \frac{36xy-70x^2}{9}=1 $$

Queda un $1$.

¿Qué pasa si pasó el 9 de la derecha restando?

$$ 36xy-70x^2-9=0 $$

Queda un $0$.